gladier_tools/tests/posix/test_untar.py

```diff
@@ -0,0 +1,29 @@
+from unittest.mock import Mock
+import tarfile, os
+import pytest
+from gladier_tools.posix.untar import untar_file
+
+@pytest.fixture
+def mock_isfile(monkeypatch):
+    """Mock isfile. Returns False by default"""
+    monkeypatch.setattr(os.path, 'isfile', Mock(return_value=False))
+    return os.path.isfile
+
+def test_no_file(monkeypatch, mock_isfile):
+    with pytest.raises(NameError):
+        untar_file()
+
+@pytest.mark.usefixtures("mock_tar")
```

Comment:
I'm not familiar with `@pytest.mark.usefixtures`, is that needed here?

Reply:
oh, I guess not. It was useful when I was using imported fixtures